ParkingLot/ParkingBoy.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace ParkingLot
+{
+    public class ParkingBoy
+    {
+        private List<ParkingLot> boysParkingLots = new List<ParkingLot>();
+        public ParkingBoy(string id)
+        {
+            this.Id = id;
+        }
+
+        public string Id { get; set; }
```

Comment:
should parkingboy's id be public?

Reply:
You are right ,set method should be private.